.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't check this file in

You ca add it to `.git/info/exclude` so that it's locally ignored

Reply:
Apologies, have removed it.